app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ActiveRecord::Base
+	has_secure_password
+
+	has_many :reviews
+	has_many :comments
+	has_many :ratings
+
+	validates :email, presence: true
```

Comment:
validates email format - google validation of email for rails and you can see a specific validation to use to make sure people input an email format